.ci/docker/common/install_magma_conda.sh

```diff
@@ -1,26 +1,23 @@
 #!/usr/bin/env bash
-# Script that replaces the magma install from a conda package
+# Script that installs magma from tarball inside conda environment.
+# It replaces anaconda magma-cuda package which is no longer published.
+# Execute it inside active conda environment.
+# See issue: https://github.com/pytorch/pytorch/issues/138506
 
 set -eou pipefail
 
-function do_install() {
-    cuda_version_nodot=${1/./}
-    anaconda_python_version=$2
+cuda_version_nodot=${1/./}
+anaconda_dir="$CONDA_PREFIX"
```

Comment:
Is $CONDA_PREFIX always set when a conda environment is enabled?

Reply:
Yes please refer to: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#macos-and-linux

Reply:
Added fallback